README.md

```diff
@@ -31,16 +31,50 @@ npm install --save-dev spectron
 Spectron works with any testing framework but the following example uses
 [mocha](https://mochajs.org):
 
+
+## To use
```

Comment:
I would suggest these headings:

To Install → Installation

To Use →  Usage

This make for nicer fragment links, like `electron/spectron#usage`